.github/workflows/build-and-publish-chart.yml

```diff
@@ -0,0 +1,96 @@
+name: build-and-publish-chart
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-publish-chart:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install Helm
+      uses: azure/setup-helm@v3
+
+    - name: Build Dependencies
+      run: |
+        helm dependency update charts/cloudzero-agent/
+
+#    - name : Update Chart Version
```

Comment:
should we remove this commented block?

Reply:
Yup we should. I'll do that before merging.